src/ui/auth/uaa/uaa.go

```diff
@@ -38,6 +39,7 @@ func CreateClient() (uaa.Client, error) {
 		ClientSecret:  UAASettings.ClientSecret,
 		Endpoint:      UAASettings.Endpoint,
 		SkipTLSVerify: !UAASettings.VerifyCert,
+		CARootPath:    os.Getenv("UAA_CA_ROOT"),
```

Comment:
Root CA is used commonly, so better to name it UAA_ROOT_CA?

Reply:
I think it's fine as it's an attribute in uaa.ClientConfig